src/apiController.js

```diff
@@ -97,87 +97,74 @@ let apiController = {
       console.error('Oh noes ', err);
     });
   },
+
+
   //Questions and Answers API calls
-  getQuestions: (productId, { pages = 1, count = 5 } = {}) => {
-    let params = { page: pages, count: count, product_id: productId };
-    return axios
-      .get('/qa/questions', { params })
+  //according to the docs the server expects one object with product_id, page, count inside as a parameter
+  getQuestions: ({ productId = 21111, page = 1, count = 5 }) => {
+    var product_id = productId;
+    let params = { product_id, page, count };
+
+    return axios.get('/qa/questions', { params })
       .then((data) => {
         return data;
       })
       .catch((err) => {
         console.error('Oh noes ', err);
       });
   },
-  getAnswers: (questionId, { pages = 1, count = 5 } = {}) => {
+  getAnswers: (questionId, { pages = 1, count = 5 }) => {
```

Comment:
I would leave the `= {}` part in, otherwise default values won't work if you don't pass in the second parameter at all. Granted, I won't be using the getAnswers api call ever, so it's up to you, but if you want to be able to call it like `apiController.getAnswers(questionId)`, you need the ` = {}`

Reply:
will change that back!